README.md

```diff
@@ -176,14 +176,25 @@ import puppeteer from 'puppeteer';
 
 **1. Uses Headless mode**
 
-Puppeteer launches Chromium in
+By default Puppeteer launches Chromium in
 [headless mode](https://developers.google.com/web/updates/2017/04/headless-chrome).
```

Comment:
How about removing this old link? The new Headless article explains everything (and points back to this post anyway).